src/codeu/chat/client/commandline/Panel.java

```diff
@@ -48,10 +48,10 @@ public void register(String commandName, Command command) {
   // will be returned. True will be return if a command is found. Whether or not
   // the command was successful is not returned.
   //
-  public boolean handleCommand(String commandName, Scanner line) {
+  public boolean handleCommand(String commandName, List<String> args) {
```

Comment:
I think you will need to import `java.util.List` for this code to compile. Also does `Command.invoke()` now accept `List<String>` ?